lib/timeline/TimeStep.js

```diff
@@ -73,7 +73,7 @@ TimeStep.FORMAT = {
     hour:       'HH:mm',
     weekday:    'ddd D',
     day:        'D',
-    week:       'w',
+    week:       'D',
```

Comment:
I don't quite understand why to choose the "Day of Month" as default instead of "Locale week of year" when local-aware moment.js is defined as a requirement. Besides, it seems to be more safe to choose the local-aware moment.js week, you never know what happens within these locales.

Reply:
The reason for that is to be consistent with the other formats, so when your zooming it doesn't jump from number of the week in the year to day of the month.
This can then be overridden to fit specific cases using ``format: { minorLabels: {week: 'w'} }`` as shown in this example: [weekStyling](http://rawgit.com/knokit/vis/feature/week-scale-dist/examples/timeline/styling/weekStyling.html)